catalogue/webapp/components/Work/Work.tsx

```diff
@@ -46,7 +46,7 @@ const Work: FunctionComponent<Props> = ({
     (work.partOf.length > 0 && work.partOf[0].totalParts)
   );
 
-  useHotjar(isArchive);
+  useHotjar(isArchive, 'view_archive');
```

Comment:
The heatmap in the hotjar dashboard has been configured to listen for a `view_archive` event.